packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -262,8 +259,8 @@ test.describe('Prop: `leadingAction`', () => {
                 } as ModalProps,
             });
 
-            const modal = page.locator(componentSelector);
-            await expect.soft(modal).toBeVisible();
+            // Follow up to remove in Jan
```

Comment:
Looks like this is overriding Josh's latest changes that went into main?

Reply:
🤦🏼